charts/keep/Chart.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v2
+name: keep
+version: 0.0.1
+description: Keep Helm Chart
+type: application
+icon: https://platform.keephq.dev/_next/image?url=%2Fkeep.png&w=48&q=75
+appVersion: 0.2.1
+deprecated: false
+annotations:
+  app: keep
+keywords:
+  - alerting
+  - monitoring
+  - monitoring-tool
+home: https://platform.keephq.dev/
+sources:
+  - https://github.com/keephq/keep
```

Comment:
change to https://github.com/keephq/helm-charts?

Reply:
done